lib/tasks/evm_dba.rake

```diff
@@ -100,37 +100,13 @@ namespace :evm do
     end
 
     desc "Seed the ManageIQ EVM Database (VMDB) with defaults"
-    task :seed do
-      Rake::Task['db:seed'].invoke
-    end
+    task :seed => "db:seed"
 
     desc "Destroys the ManageIQ EVM Database (VMDB) of all tables, views and indices"
-    task :destroy do
-      begin
-        Rake::Task['environment'].invoke
-      rescue => err
-        # Allow "destroying" a database that doesn't exist
-        raise unless err.message =~ /does not exist$/
-      end
-
-      Rake::Task['db:drop'].invoke
-      Rake::Task['db:create'].invoke
-
-      # db:create creates a temporary connection to the default database, but doesn't
-      # remove the connection in the event of a failed create, so we drop the connection
-      # and reestablish it to the environment's database.
-      ActiveRecord::Base.remove_connection
```

Comment:
Do we still need this logic?  Maybe git blame it and see if it's still needed?

Reply:
Right, but did you verify that the concerns here aren't valid concerns anymore?

Reply:
1. `Rake::Task['environment'].invoke` 87ed9bc9dd7
   introduced to avoid error when resetting a non-existing database. works
2. `Rake::['db:drop'].invoke` (and `:create`) fa37b84c
   introduced for SqlServer. No longer using SqlServer (Postgres always worked w/ original)
   this is actually rolling back to the original code pre SqlServer

Reply:
moved into #22003